### ISSUE TITLE
Chop indicator value is different from the Trading view

### ISSUE THREAD

**Issue by nightrod** [twopirllc/pandas-ta#214]
The value of the indicator is different because you are using LOG10 instead of the natural logarithm.
Although Math.log10 is also listed in the source you provided, it is possible that it is replaced by the original ln (x)
JavaScript also changes the name of the ln () function to log (), and mathematically log () is log10 ()

It is possible that log in chopindex Tvw is wrong, but the question is who is the authority here

**Comment by twopirllc** [comment]
Hello @nightrod,

> It is possible that log in chopindex Tvw is wrong, but the question is who is the authority here

Unfortunately, there is no official authority on Technical Analysis. However since TA Lib is the de facto library used, that I have seen, I have made it the authority. For **chop**, there is no corresponding indicator in the library, so I have chose TradingView.

> The value of the indicator is different because you are using LOG10 instead of the natural logarithm.

Thanks for the insight on Javascript and log vs ln. Feel free to _fork_, _edit_, and make a _PR_ if you want this resolved in a timely manner. Your contribution _would_ be greatly appreciated. Make sure the user has the option to choose _log10_ and _ln_.

Kind Regards,
KJ
